Source/Core/Core/AchievementManager.cpp

```diff
@@ -1020,8 +1037,15 @@ void AchievementManager::FetchBadge(AchievementManager::Badge* badge, u32 badge_
 
     if (!LoadPNGTexture(badge, *http_response))
     {
-      ERROR_LOG_FMT(ACHIEVEMENTS, "Default game badge '{}' failed to load",
```

Comment:
Unrelated to the rest of the change here but I happened to notice this log was wrong so I threw it in.